src/lookupDataProvider.js

```diff
@@ -7,6 +7,12 @@ class LookupDataProvider extends Component {
         lookupData: null
     }
 
+    async componentDidMount() {
+        const data = await lookupTablesService.getAllLookupData();
+        this.setState({
+            lookupData: data.data
+        });
+    }
     render() {
         return (
             <LookupDataContext.Provider value={{
```

Comment:
Je dobrym pravidlom do `value` posielat len referenciu na objekt a nevytvarat ho tak, ako to je tu. pretoze pri kazdom re-renderi parent komponenty providera sa vytvori novy objekt, co sposobi re-render vsetkych consumerov. Popisane to je tu https://reactjs.org/docs/context.html#caveats.

V tomto pripade to nemusi znamenat problem, ale odporucam to zapisat takto:

```js
<LookupDataContext.Provider value={this.state}>	
  {this.props.children}
</LookupDataContext.Provider>
```

Reply:
done